book_source/06_reference/04_docker/05_building_images.Rmd

```diff
@@ -0,0 +1,141 @@
+### Building and modifying images {#docker-build-images}
+
+For general use, it is sufficient to use the pre-built PEcAn images hosted on [Docker Hub](https://hub.docker.com/r/pecan/).
+However, there are cases where it makes sense to re-build the Docker images locally.
```

Comment:
right here, I am looking for the 'quick-start' command to build / deploy the entire stack (is it elsewhere or can it be here?), e.g. 

```
docker build -t pecan/depends:latest -f docker/base/Dockerfile.depends .
```

Reply:
I don't really think that's necessary since that is a previous section in the same Docker chapter. But, as requested, I added it anyway (833ffeca8).